src/web/components/SharingPermission/ParticipantTableHelper.ts

```diff
@@ -16,6 +16,15 @@ export const isAddedByManual = (site: SharingSiteWithSource) => {
   return site.addedBy.includes(MANUALLY_ADDED);
 };
 
+export const disableSelectAllCheckbox = (sites: SharingSiteWithSource[]) => {
+  for (const site of sites) {
```

Comment:
nit: I think disableSelectAll is true when none of the site is added by manual? maybe we could use `sites.every(site => !isAddedByManual(site))` or `!sites.some(isAddedByManual)`

Reply:
nice suggestions, went with `!sites.some(isAddedByManual)`.